bin/release_android

```diff
@@ -17,22 +17,41 @@ if [ -z "$VERSION" ]; then
     exit 1
 fi
 
-USER=$2
-if [ -z "$USER" ]; then
-    print_error "Missing Bintray API user"
-    exit 1
-fi
+URL="https://oss.sonatype.org/service/local/staging/deploy/maven2"
 
-TOKEN=$3
-if [ -z "$TOKEN" ]; then
-    print_error "Missing Bintray API token"
-    exit 1
+if [ "$VERSION" == "0.0.0-dev" ]; then
+    VERSION="0.0.0-SNAPSHOT"
+    URL=https://oss.sonatype.org/content/repositories/snapshots
```

Comment:
URL should be injected from Travis, instead of of trying to guess it from version

Reply:
In this case, we will guess it in Travis. I just wanted to keep related things together.